crates/re_data_source/src/data_loader/mod.rs

```diff
@@ -2,10 +2,106 @@ use std::sync::Arc;
 
 use once_cell::sync::Lazy;
 
-use re_log_types::{ArrowMsg, DataRow, LogMsg};
+use re_log_types::{ArrowMsg, DataRow, EntityPath, LogMsg, TimePoint};
 
 // ---
 
+/// Recommended settings for the [`DataLoader`].
```

Comment:
I get the `recommended` method on this, but I'm confused/not convinced that this thing as a whole is "recommended settings". Why not just `DataLoaderSettings` and then each field elaborates what effect it expects (already the case)? I get that we're defensive here and want to make clear that a data loader that ignores the settings is still a valid data loader and these are all "would you kindly please"-arguments, but I find baking that into the name goes a bit too far and is more confusing than helping.

... anyways, not a hill I gonna die on so if you prefer this as is keep it :)

Reply:
I wonder if there's something more we could write about the (lack of) consequences if a data loader ignores the args

Reply:
Sure we can rename to `DataLoaderSettings`.

> I wonder if there's something more we could write about the (lack of) consequences if a data loader ignores the args

We've already done so in a bunch of places, including the guide.